newsSummaryView.test.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const fs = require('fs');
+const NewsSummaryModel = require('./newsSummaryModel');
+const NewsSummaryView = require('./newsSummaryView');
+
+let newsModel;
+let newsView;
+let fakeApi;
+
+beforeEach(() => {
+  document.body.innerHTML = fs.readFileSync('./index.html');
+
+  fakeApi = { getHeadlines: (callback) => callback(
```

Comment:
So my idea was to use the beforeEach() to avoid repeating the same lines of code in multiple it blocks, but I found out that if I declared the variables within the beforeEach() block they would be only visible in the scope of that block. Therefore I used the solution of declaring them before the beforeEach block (which I saw in some threads online). Was it a bad idea?